search/tutorials/tutorial_ordering.md

```diff
@@ -1,136 +1,216 @@
 
-and change the ordering expression to the following one:
+We have prepared a JSON file with valid products in the `search` directory:
 
-```order = 'price asc'``` or just ```order = 'price'``` - those are equivalent expressions since ascending is the default ordering direction.
+**search/products_for_search.json**
 
-Run the sample in a terminal using the command:
+You can use this file in the tutorial.
+
+1.  In your own Google Cloud project, go to Cloud Storage.
+1.  Click **Create bucket**, give your Project ID in the name, and
+    click **Create**.
+1.  From the Cloud Shell Terminal, run the following command:
+    ```bash
+    gsutil cp search/products_for_search.json gs://<YOUR_PROJECT_ID>
+    ```
+
+1.  Now you can see that the file is uploaded to the Cloud Storage bucket.
+
+### Import products to the Retail Catalog
+
+To import the prepared products to a catalog, run the following command in the Terminal:
 
 ```bash
-python search_with_ordering.py
+python product/import_products_gcs.py
 ```
 
-You have sorted the products by price ascending.
+## Order by a single field: ordering expression
+
+To use the ordering feature, you need to specify the field and the ordering direction. You can order by both the text and numeric fields.
+
+1. Open <walkthrough-editor-select-regex filePath="cloudshell_open/grs-samples-python/search/search_with_ordering.py" regex="TRY DIFFERENT ORDERING EXPRESSIONS HERE">search_with_ordering.py</walkthrough-editor-select-regex>.
+
+1. Order the search results by price when more expensive items come first. To do that, set the ordering expression as follows:
+
+    ```order = 'price desc'```
+
+
+1. Run the sample in the Terminal using the following command:
+    ```bash
+    python search/search_with_ordering.py
+    ```
+
+As you can see, the results are now ordered by descending price.
+
+## Order by a single field: product sorting
+
+Next, change the ordering direction to show the cheapest products first.
 
-## Ordering by multiple fields
+1. Change the condition under the <walkthrough-editor-select-regex filePath="cloudshell_open/grs-samples-python/search/search_with_ordering.py" regex="TRY DIFFERENT ORDERING EXPRESSIONS HERE">comment</walkthrough-editor-select-regex> to the following:
 
-You can order items by multiple fields using the comma-separated fields in order of priority (more prioritized come first). 
+    ```order = 'price asc'``` or just ```order = 'price'```
+    Those are equivalent expressions because ascending is the default ordering direction.
 
-To order items with equal values for higher priority fields, use the lower priority fields. 
+1. Run the sample in the Terminal using the command:
+
+    ```bash
+    python search/search_with_ordering.py
+    ```
+
+You have sorted the products by ascending price.
+
+## Order by multiple fields
+
+You can order items by multiple fields using the comma-separated fields in order of priority, with fields having higher priority coming first.
```

Comment:
You can order items by multiple fields using the comma-separated fields in order of priority, with the higher priority fields coming first.